server/real_environment/real_environment.go

```diff
@@ -85,6 +88,7 @@ func NewRealEnv(c *config.Configurator, h interfaces.HealthChecker) *RealEnv {
 		healthChecker: h,
 
 		executionClients: make(map[string]*executionClientConfig, 0),
+		mux:              tracing.NewHttpServeMux(http.NewServeMux()),
```

Comment:
nit: we should keep the environment pretty agnostic to stuff (like, whether or not the mux is a tracing mux or regular mux). Could you add a Set method in env for the mux and then set it in libmain when you create a new env?

Reply:
Fixed.